src/TheThingsNetwork.cpp

```diff
@@ -237,8 +236,10 @@ bool TheThingsNetwork::personalize() {
 bool TheThingsNetwork::provision(const byte appEui[8], const byte appKey[16]) {
   sendCommand(F("mac set appeui"), appEui, 8);
   sendCommand(F("mac set appkey"), appKey, 16);
-  sendCommand(F("mac save"));
-  return waitForOK();
+  while (!sendCommand(F("mac save"))) {
+    delay(1000);
+  }
+  return waitForOK(10000);
```

Comment:
@FokkeZB I tested it join() works for me, that is what I got: 
```
Version is RN2483 1.0.1 Dec 15 2015 09:38:09, model is RN2483
Sending: mac set deveui 0004A30B001B4C9F
Sending: mac set adr off
Sending: mac set pwridx 1
Sending: mac set dr 5
Sending: mac set appeui with 8 bytes
Sending: mac set appkey with 16 bytes
Sending: mac save
Wait for OK time-out expired
Sending: mac save
Sending: mac set deveui 0004A30B001B4C9F
Sending: mac join otaa
Join accepted. Status: 00000401
EUI: 0004A30B001B4C9F
Battery: 3315
AppEUI: 70B3D57ED0000D35
DevEUI: 0004A30B001B4C9F
DevAddr: 1D1FF5E3
Band: 868
Data Rate: 5
RX Delay 1: 1000
RX Delay 2: 2000
```
For some reason ``` mac save ``` doesn't work the first time.